packages/file_selector/file_selector/README.md

```diff
@@ -27,11 +28,23 @@ or read/write access:
 ```
 depending on your use case.
 
-### Examples
+# Feature supported platform
+Features support by platform:
+
+| Feature              | iOS      | Linux     | macOS     | Windows     | Android  |
+| -------------------- |--------- | --------- | --------- | ----------- | -------- |
+| Open a single file   | ✔️       | ✔️        | ✔️       | ✔️          | ❌      |
```

Comment:
I think it would be clearer to describe all of these, other than save as "Choose"; it doesn't actually open anything. (The API naming isn't ideal, but we don't need to replicate it here.)